pybrake/test_routes.py

```diff
@@ -0,0 +1,74 @@
+import time
```

Comment:
Almost all tests don't have asserts. Is that expected?

Reply:
The majority of test cases have been updated, although some changes are likely.

Reply:
Sorry, I am not following. Do you mean that some test cases purposefully don't have an assertion?